packages/@react-spectrum/s2/test/SelectBoxGroup.test.tsx

```diff
@@ -0,0 +1,787 @@
+          <SelectBox value="option1">
+            <Text slot="label">Option 1</Text>
+          </SelectBox>
+          <SelectBox value="option2">
+            <Text slot="label">Option 2</Text>
+          </SelectBox>
+        </SelectBoxGroup>
+      );
+
+      const listbox = screen.getByRole('listbox');
+      await act(async () => {
+        listbox.focus();
+      });
+      
+      // Navigate to second option
+      await userEvent.keyboard('{ArrowDown}');
+      
+      // Check that navigation works by verifying an option has focus
+      const option1 = screen.getByRole('option', {name: 'Option 1'});
+      expect(option1).toHaveFocus();
+    });
+  });
+
+  describe('Children validation', () => {
+    let consoleSpy: jest.SpyInstance;
+
+    beforeEach(() => {
+      consoleSpy = jest.spyOn(console, 'warn').mockImplementation(() => {});
+    });
+
+    afterEach(() => {
+      consoleSpy.mockRestore();
+    });
+
+    it('does not warn with valid number of children', () => {
+      render(
+        <SelectBoxGroup 
+          aria-label="Valid children test" 
+          selectionMode="single" 
+          onSelectionChange={() => {}} 
+          selectedKeys={new Set()}>
+          <SelectBox value="option1">
+            <Text slot="label">Option 1</Text>
+          </SelectBox>
+          <SelectBox value="option2">
+            <Text slot="label">Option 2</Text>
+          </SelectBox>
+        </SelectBoxGroup>
+      );
+      
+      expect(console.warn).not.toHaveBeenCalled();
+    });
+  });
+
+  describe('Accessibility', () => {
+    it('has proper listbox structure', () => {
+      render(
+        <SelectBoxGroup 
+          aria-label="Accessibility test"
+          selectionMode="single" 
+          onSelectionChange={() => {}} 
+          selectedKeys={new Set()}>
+          <SelectBox value="option1">
+            <Text slot="label">Option 1</Text>
+          </SelectBox>
+          <SelectBox value="option2">
+            <Text slot="label">Option 2</Text>
+          </SelectBox>
+        </SelectBoxGroup>
+      );
+      
+      expect(screen.getByRole('listbox')).toBeInTheDocument();
+      expect(screen.getAllByRole('option')).toHaveLength(2);
+    });
+
+    it('supports aria-label and aria-labelledby', () => {
+      render(
+        <div>
+          <h2 id="test-label">My SelectBoxGroup</h2>
+          <SelectBoxGroup 
+            aria-labelledby="test-label"
+            selectionMode="single" 
+            onSelectionChange={() => {}} 
+            selectedKeys={new Set()}>
+            <SelectBox value="option1">
+              <Text slot="label">Option 1</Text>
+            </SelectBox>
+          </SelectBoxGroup>
+        </div>
+      );
+      
+      const listbox = screen.getByRole('listbox');
+      // Just verify the listbox has an aria-labelledby attribute  
+      expect(listbox).toHaveAttribute('aria-labelledby');
+      expect(listbox.getAttribute('aria-labelledby')).toBeTruthy();
```

Comment:
hm, seems to intentionally only check for the existence of `aria-labelledby`, but as noted above the SelectBoxGroup should accept a user defined `aria-labelledby`/`aria-label`